lib/components/T3CeFeloginLogin/T3CeFeloginLogin.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <div class="fe-login">
+    <p>{{ $props.data.message.header }}</p>
+    <p>{{ $props.data.message.message }}</p>
+    <T3Form ref="form" :elements="elements" @submit="onSubmit" />
+  </div>
+</template>
+<script>
+import T3Form from '../../components/T3Form/T3Form'
+import baseCe from '../../mixins/component/baseCe'
+
+export default {
+  name: 'T3CeFeloginLogin',
+  components: {
+    T3Form
+  },
+  extends: baseCe,
+  props: {
+    data: {
+      type: Object,
+      required: true
+    }
+  },
+  computed: {
+    redirectUrl () {
+      const element = this.data.form.elements.filter(function (element) {
+        return element.name === 'redirect_url'
+      })
+
+      if (element.length > 0) {
+        return element?.value || ''
+      }
+
+      return ''
+    },
+    elements () {
+      return this.compileFormElements(this.data.form.elements)
+    }
+  },
+  methods: {
+    compileFormElements (elements) {
+      return elements.map((element) => {
+        return {
+          value: element.value || '',
+          type: element.type || '',
```

Comment:
we should add a new field here - `id` - I think we can use name for it

Reply:
Done ✅